neptune/new/attributes/atoms/string.py

```diff
@@ -24,7 +24,7 @@
 from neptune.new.types.atoms.string import String as StringVal
 
 if typing.TYPE_CHECKING:
-    from neptune.new.attribute_container import AttributeContainer
+    from neptune.new.attributes_containers import AttributeContainer
```

Comment:
I have to admit: plural "attributes containers" does sound BAD for me (vs attribute container)